components/rhs_thread/rhs_thread.tsx

```diff
@@ -345,7 +361,7 @@ export default class RhsThread extends React.Component {
                 <FloatingTimestamp
                     isScrolling={this.state.isScrolling}
                     isMobile={Utils.isMobile()}
-                    postId={this.state.topRhsPostId}
+                    createAt={new Date(Date.now())}
```

Comment:
Why is this changing? This also looks incorrect since it seems like it would make the floating timestamp (only visible when you shrink the window enough for the app to switch to mobile view) show the current date instead of the date of the post at the top of the screen

Reply:
Thanks for fixing all those types. This comment still needs another look though.

The `FloatingTimestamp` doesn't actually need to have `createAt` passed into it since the component gets that based on the `postId` that's passed in. The logic for that is [here](https://github.com/mattermost/mattermost-webapp/blob/master/components/post_view/floating_timestamp/index.js). This can be reverted.

Reply:
The thing is, `createAt` is a required prop on `FloatingTimestamp`